vpc-terraform/ansible/playbook/nginx-role/tasks/main.yml

```
@@ -0,0 +1,15 @@
+  - name: Install nginx 
+    apt: 
+      name: nginx
+      state: latest
+
+  - name: Add html file
+    copy:
+      src: index.html
+      dest: /var/www/html
+
+  - name: Enable nginx
+    service :
```

yamllint (1.35.1)

[warning] too many spaces before colon

(colons)